README.md

```diff
@@ -57,7 +57,7 @@ to a wider set of architectures and environments, or compliance requirements.  S
 Specifying `default-features = false` when depending on rustls will remove the
 dependency on aws-lc-rs.
 
-Rustls requires Rust 1.63 or later.
+Rustls requires Rust 1.63 or later. It has an optional dependency on zlib-rs which requires 1.75 or later.
```

Comment:
Maybe these changes should be in the commit that introduces the use of zlib-rs? I think it might be nice to also make explicit that the zlib-rs dependency is "optional (and disabled by default)".

Reply:
Squashed this commit into that one.